server/services/TransferService.js

```diff
@@ -107,6 +187,13 @@ class TransferService {
     try {
       await this._session.beginTransaction();
 
+      log.info(transferId, walletLoginId);
```

Comment:
log.debug, also don't think it is needed here

Reply:
Removed